tests/ffi/qm-oom-score-adj/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+# Copy container image registry to /var/qm/lib/containers
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+if [ ! -d "${QM_HOST_REGISTRY_DIR}" ]; then
+    exec_cmd "mkdir -p ${QM_HOST_REGISTRY_DIR}"
+    exec_cmd "podman push ${image_id} dir:${QM_HOST_REGISTRY_DIR}/tools-ffi:latest"
```

Comment:
IIUC this is 1.5GB. taht's going to take a while for no good reason. Unless there is a plan to use the same container inside QM for other test of course

Reply:
@Yarboa  don't we have the version with the `--build-arg shortbuild=yes ` available  somewhere? 

From:
https://gitlab.com/CentOS/automotive/container-images/ffi-tools/

```
Thin image:
In case of testing only, add the following --build-arg shortbuild=yes to build command
# podman build -f Containerfile --build-arg shortbuild=yes --cap-add=sys_admin -t localhost/ffi-tools:latest .
```

Reply:
@dougsland @Yarboa Is there any problem to use an alpine:latest image instead the tool-ffi:latest?

Reply:
@odra for the shortversion question.

Reply:
@telemaco I am always in favor of eating our own food but I understand your concern. I was thinking here, should we change the logic:  thin version as default and make the **fat** version as the optional?  
@odra @Yarboa do you think we can make this happen?

Reply:
Actually, everything is ready for that 
Opened an issue,  
https://gitlab.com/CentOS/automotive/container-images/ffi-tools/-/issues/2

Will work on that
Anyway, once tests are running one after each other, unless qm reboot container persist for other tests of qm

Reply:
1.5GB is a lot better than 3.5 GB, but it still leaves a lot to be desired. I think generally the test duration would greatly benefit of some generic optimizations (limiting number of layers, cleanup of caches) - none of that is done ATM so now wonder these images are monstrous.
It takes MINUTES to copy such amount of data on weaker boards

Reply:
@michalskrivanek you are correct,
related to (limiting number of layers, cleanup of caches)
this one was not defined yet, 
Currently we are running c9s and other SoC which are more powerfull.

I advise to open a detailed ticket for that, with tests definitions

Reply:
Will be always things to improve, lets merge this one and continue this request in parallel.
https://github.com/containers/qm/issues/447